backend/enhanced_guide_generator.py

```
@@ -0,0 +1,796 @@
+                "medical_facilities": "Excellent healthcare system, travel insurance recommended",
+                "safety": "Generally very safe, standard precautions in tourist areas",
+                "pharmacies": "Green cross signs, some open 24/7"
+            },
+            "communication": {
+                "wifi": "Widely available in hotels, restaurants, and public areas",
+                "mobile": "EU roaming rates apply, local SIM cards available",
+                "useful_apps": ["Citymapper", "Google Translate", "OpenTable"]
+            }
+        }
+    
+    def _generate_weather_forecast(self, destination: str, start_date: str, end_date: str) -> Dict:
```

Ruff (0.13.1)

Unused method argument: `destination`

(ARG002)